ajax/libs/readremaining.js/0.1.0/src/readremaining.jquery.js

```diff
@@ -0,0 +1,268 @@
+/*!
+ * ReadRemaining.js 1.0.0
```

Comment:
The author had modified it, and I also had updated it. @x09326 Please help me review again, thank you.